CONTRIBUTING.md

```diff
@@ -40,8 +40,8 @@ appropriate for programmer's failures on the Dart side of things. But time will
 
 #### Debugging the app from launch
 
-- Android: Go into developer options and use the "Wait for debugger" option. Launch the app from the Flutter project (not the `android` directory) in Android Studio. Then in the Android project in Android Studio, attach the debugger.
-- iOS: In Xcode, build the Xcode app scheme with launch option: "Wait for the executable to be launched" instead of "Automatically". You need to repeat this for every launch. Then "Attach Flutter" in Android Studio or CLI if you want to, as well. 
+- Android: Go into developer options and use the "Wait for debugger" option. Launch the app from the Flutter project (not `android` directory) in Android Studio. This ensures the `--dart-define` variables are used by the application. Then in the Android project in Android Studio, attach the debugger. This allows you to debug Java code, such as the message handlers. Then "Attach Flutter" in Android Studio or CLI if you want to, as well. 
```

Comment:
Screenshots here would be really useful, IMO!

Reply:
I've moved this to the user facing guide (PushNotifications.md) instead of the contributing guide, because it would be useful for them too. IMHO, because of this, it makes sense to add the screenshot c31121cdc961f8cc3bba76d2017b47a7e548c9cc

If this section was kept in the contributing guide, then I think the cost of generating the screenshots is greater than the benefit, since Xcode schemes and developer options are commonly used tools.

Reply:
You can see the preview at https://github.com/ably/ably-flutter/blob/feature/push-notifications-documentation/PushNotifications.md

Reply:
I think this is a very sensible way of approaching this.